web/html/src/manager/admin/list-payg/list-payg.tsx

```diff
@@ -0,0 +1,205 @@
+
+import * as React from "react";
+import { useEffect } from "react";
+
+import _truncate from "lodash/truncate";
+
+import PaygStatus from "manager/admin/payg-shared/common/payg-status";
+
+import { Button } from "components/buttons";
+import { FromNow } from "components/datetime/FromNow";
+import withPageWrapper from "components/general/with-page-wrapper";
+import { ServerMessageType } from "components/messages";
+import { SectionToolbar } from "components/section-toolbar/section-toolbar";
+import { Column } from "components/table/Column";
+import { SearchField } from "components/table/SearchField";
+import { Table } from "components/table/Table";
+import { showSuccessToastr } from "components/toastr/toastr";
+import { HelpLink } from "components/utils";
+
+import { Utils } from "utils/functions";
+
+const _SETUP_WIZARD_STEPS = [
+  {
+    id: "wizard-step-proxy",
+    label: "HTTP Proxy",
+    url: "/rhn/admin/setup/ProxySettings.do",
+    active: false,
+  },
+  {
+    id: "wizard-step-credentials",
+    label: "Organization Credentials",
+    url: "/rhn/admin/setup/MirrorCredentials.do",
+    active: false,
+  },
+  {
+    id: "wizard-step-suse-products",
+    label: "Products",
+    url: "/rhn/manager/admin/setup/products",
+    active: false,
+  },
+  {
+    id: "wizard-step-suse-payg",
+    label: "Pay-as-you-go",
+    url: "/rhn/manager/admin/setup/payg",
+    active: true,
+  },
+];
+
+type PaygOverviewType = {
+  id: String;
+  host: String;
+  description: String;
+  status: String;
+  statusMessage: String;
+  lastChange: moment.Moment;
+};
+
+type Props = {
+  payg_instances: Array<PaygOverviewType>;
+  flashMessage?: ServerMessageType;
+};
+const ListPayg = (props: Props) => {
+  useEffect(() => {
+    if (props.flashMessage) {
+      showSuccessToastr(props.flashMessage);
+    }
+  }, []);
+
+  const searchData = (row, criteria) => {
+    const keysToSearch = ["host", "description", "status"];
+    if (criteria) {
+      return keysToSearch
+        .map((key) => row[key])
+        .join()
+        .toLowerCase()
+        .includes(criteria.toLowerCase());
+    }
+    return true;
+  };
+
+  const title = (
+    <div className="spacewalk-toolbar-h1">
+      <h1>
+        <i className="fa fa-cogs"></i>
+        &nbsp;
+        {t("Setup Wizard")}
+        &nbsp;
+        <HelpLink url="reference/admin/setup-wizard.html" />
+      </h1>
+    </div>
+  );
+
+  const tabs = (
+    <div className="spacewalk-content-nav">
+      <ul className="nav nav-tabs">
+        {_SETUP_WIZARD_STEPS.map((step) => (
+          <li key={step.id} className={step.active ? "active" : ""}>
+            <a className="js-spa" href={step.url}>
+              {t(step.label)}
+            </a>
+          </li>
+        ))}
+      </ul>
+    </div>
+  );
+
+  let pageContent = (
+    <Table
+      data={props.payg_instances}
+      identifier={(row) => row.host}
+      initialSortColumnKey="name"
+      initialItemsPerPage={window.userPrefPageSize}
```

Comment:
Specifying `initialItemsPerPage` with the default value is not needed (https://github.com/uyuni-project/uyuni/pull/4617).

Reply:
done. thanks